libs/designsystem/src/lib/components/avatar/avatar.component.scss

```diff
@@ -135,7 +135,7 @@ $badge-diameter: $avatar-badge-size;
 @each $color-name,
   $color-value
     in map-merge(
-      $main-colors,
+      $_color-palette,
```

Comment:
This effectively also adds classes/support for:
* `background-color`
* `black`
* `semi-dark`

otherwise removed from `$_color-palette` in `$main-colors` map... Is this intentionally?

Reply:
For the change here we just want to add semi-light

Reply:
Is it a problem to add support for these other colors?

Reply:
It might be - UX usually have a very strong opinion on what to use and when...